src/System.Runtime.WindowsRuntime/ref/System.Runtime.WindowsRuntime.csproj

```diff
@@ -23,10 +23,6 @@
     <Compile Include="System.Runtime.WindowsRuntime.Manual.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ReferencePath Include="$(ExternalAPIsPath)\NetFX\Contracts\DesignTimeFacades\mscorlib.dll" />
```

Comment:
You didn't need mscorlib?  I thought the projections emitted refs to mscorlib, not System.Runtime.